docs/frameworks.md

```diff
@@ -30,6 +30,7 @@ layout: documentation
 
 * [Chronos](https://github.com/mesos/chronos) is a distributed job scheduler that supports complex job topologies. It can be used as a more fault-tolerant replacement for Cron.
 * [Cook](https://github.com/twosigma/cook) is a job scheduler like Torque that not only supports individual tasks, but also Spark. Cook provides powerful automatic preemption and multitenancy features for shared clusters, in order to guarantee throughput to all users while allowing individuals to temporarily "burst" to additional resources as needed. Cook provides a simple REST API & Java client for interaction.
+* [Elastic-Job-Cloud](https://github.com/dangdangdotcom/elastic-job) is a distributed scheduled job cloud solution. It focus on scale out, fault-tolerant(failover, misfire), flexible job scenarios(transient, daemon), complex trigger(cron, event driven), extra job templates(dataflow), jobs running history trace.
```

Comment:
How about "is a distributed scheduled job cloud solution designed with HA and fault-tolerance in mind. It focuses on horizontal scaling, and provides transient and daemon jobs, event and schedule based job triggers, job dependencies, and job history."

I tried to rephrase this a bit, does it still sound accurate?